scripts/azure-linux_mac.yml

```diff
@@ -134,12 +140,26 @@ steps:
       docker ps -a
     fi
 
+    if [[ ( "$AGENT_OS" == "Linux" && "$TILEDB_AZURE" == "ON" ) ]]; then
+      # make sure docker is still running...
+      printenv
+      docker ps -a
+    fi
+
     make -j4 -C tiledb tiledb_unit
 
     if [[ "$TILEDB_CI_ASAN" == "ON" ]]; then
       export ASAN_OPTIONS=detect_leaks=0 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libasan.so.4
     fi
-    make check
+
+    # We do not support Azurite on OSX. When Azure is enabled, skip the azure
```

Comment:
Not necessary right away, but I think Azurite is pure javascript, so we could install and run it under node (which I believe is available by default on azure).

Reply:
Nevermind, I see it's not brew availability but that azurite has unfixed bugs on OSX.

Reply:
Actually I thought this was the first: brew availability. I just didn't want to immediately deal with `npm`. I suspect I probably have a poorly-worded comment in this patch that you may have interpreted as me saying there are OSX-specific bugs -- is that true or did you find OSX-specific bugs elsewhere?

Reply:
I misunderstood the open macOS bug I saw, which doesn't affect storage.

Sounds good for now. I believe node is available on all images, so in the future we might be able to run on all platforms w/out docker.